.github/workflows/pr-actions.yml

```diff
@@ -59,7 +59,7 @@ jobs:
     runs-on: ubuntu-latest
     needs: test-unit
     # stop the job if it runs over 20 minutes to prevent a hanging process from eating minutes
-    timeout-minutes: 20
+    timeout-minutes: 45
```

Comment:
I wish I never this this as low as I did, lol.

Reply:
Lol